src/components/UpdateJournalEntryModal/UpdateJournalEntryModal.css

```diff
@@ -0,0 +1,16 @@
+.update-entry-modal {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.update-modal-container {
+    max-width: 400px;
+    padding: 30px;
+    background-color: white;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
```

Comment:
Need to edit layout of this modal. The form is pretty long, taking up the whole height of my laptop. Should look into layout/design so that its more user friendly when on different screen sizes or devices